src/dir_walker.rs

```diff
@@ -175,6 +175,23 @@ fn ignore_file(entry: &DirEntry, walk_data: &WalkData) -> bool {
         return true;
     }
 
+    // Entry is inside an ignored absolute path
```

Comment:
Let's pull this function out to a separate method and combine it with the call on line 130:

`let is_ignored_path = walk_data.ignore_directories.contains(&entry.path());`